smokey/src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -117,7 +117,7 @@ public class DrivetrainSubsystem extends SubsystemBase {
   private final SwerveModule backLeftModule;
   private final SwerveModule backRightModule;
 
-  private SwerveDriveOdometry swerveOdometry = null;
+  private SwerveDrivePoseEstimator swervePoseEstimator = null;
```

Comment:
Need to learn more about what this one does.  Does it have SwerveDriveOdometry baked into it?  Does it subclass SwerveDriveOdometry by chance?

Reply:
Will do more later when I get some time to read up on what SwerveDrivePoseEstimator does.

Reply:
The documentation is here: https://first.wpi.edu/wpilib/allwpilib/docs/release/java/edu/wpi/first/math/estimator/SwerveDrivePoseEstimator.html

"This class wraps Swerve Drive Odometry to fuse latency-compensated vision measurements with swerve drive encoder distance measurements. It is intended to be a drop-in replacement for SwerveDriveOdometry."

Reply:
cool ... thanks ... get it